humanitarian_feedback/dashboard_flexmonster/templates/dashboard_with_pivot.html

```diff
@@ -0,0 +1,60 @@
+<head>
+  <meta charset="UTF-8">
+  <title>Dashboard with Flexmonster</title>
+  <script src="https://cdn.flexmonster.com/flexmonster.js"></script>
+  <script src="https://code.jquery.com/jquery-3.3.1.min.js"></script>
+  <link rel="stylesheet" href="https://cdn.flexmonster.com/demo.css">
+</head>
+<body>
+<!-- TODO: Somehow I need to pass pivot_data url / function the survey_id parameter -->
+<div id="pivot-table-container" data-url="{% url 'pivot_data' survey_id=4%}"></div>
```

Comment:
@AlexxxH I think this is the line I need to fix now - somehow I need to pass `survey_id` through and doing it here works, but is obviously hard-coded the survey ID - is there way to access this parameter at this point or some alternative approach I should use?

Reply:
Ah, I think I might have understood what I was missing (or at least one way to fix this): I should pass `survey_id` as a parameter to the overall template via this line: https://github.com/THEPortatCERN/Hackathon2020-Pier84/pull/1/files#diff-89918cd6e497bfe3ee6e33b23d2ef640R11